client/src/components/Projects/index.jsx

```diff
@@ -9,4 +9,4 @@ export default function Projects() {
       <Link to={'/project/2/1'}>project 2</Link>
     </div>
   );
-}
+}
```

Comment:
:thinking: 
why you commit this change?

Reply:
cuz remove my old change